en/orm/behaviors/counter-cache.rst

```diff
@@ -117,8 +131,26 @@ then updates the counter of the *previously* associated item.
     behavior to the ``CommentsTable`` in order to generate ``comment_count`` for
     Articles table.
 
-    It is possible though to make this work for ``belongsToMany`` associations.
-    You need to enable the CounterCache behavior in a custom ``through`` table
-    configured in association options and set the ``cascadeCallbacks`` configuration
-    option to true. See how to configure a custom join table
-    :ref:`using-the-through-option`.
+Belongs to many Usage
+==============
```

Comment:
Underline length should match the text length.

Reply:
damn, again that error 🙈